readme.md

```diff
@@ -112,23 +112,23 @@ instanceC.toPlural(); // money (no change)
 
 * **Verb conjugation**
 	1. Dictionary lookup (known irregularities + 4000 common verbs)
-	2. If the passed verb is identified as infinitive then applies regular expression transformations that are based on word endings, vowels and consonant phonetics.
+	2. If the passed verb is identified as infinitive it then applies regular expression transformations that are based on word endings, vowels and consonant phonetics.
 	3. Tries to trim character from the beginning of the verb, thus solving prefixes (e.g. undergoes, overthrown)
 	4. Tries to stem the word and get the infinitive form, then apply regular expression transformations.
 	5. Applies regular expressions.
 
 
 ## How accurate is it?
 
-First of all, unless you have a dictionary of all the words and verbs there are in the English you can't really write a regular expression or an algorithm and expect to have a 100% success rate. English has been adopting words form a lot of different languages (French, Greek and Latin for example) and each one these languages has it's own rules of pluralization and singularization let alone verb conjugation.
+First of all, unless you have a dictionary of all the words and verbs that exist in English you can't really write a regular expression or an algorithm and expect to have a 100% success rate. English has been adopting words from a lot of different languages (French, Greek and Latin for example) and each one of these languages has its own rules of pluralization and singularization, let alone verb conjugation.

```

Comment:
Should insert a comma after "English" in the first sentence and before "and each one" in the second.